index.html

```diff
@@ -84,197 +46,249 @@ <h4>Zero-ops OpenStack on just about any Linux box.</h4>
-          </p>
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="snap refresh --channel=latest/beta microstack" value="sudo snap refresh --channel=latest/beta microstack" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
-          <p>
-            or
-          </p>
-          <div class="p-code-copyable">
-            <input class="p-code-copyable__input" aria-label="snap refresh --channel=1.11/stable microstack" value="sudo snap refresh --channel=1.11/stable microstack" readonly="readonly">
-            <button class="p-code-copyable__action">Copy to clipboard</button>
-          </div>
-          <p>
-            to get the required version.
-          </p>
-          <p>
-            For more information refer to the <a class="p-link--external" href="https://opendev.org/x/microstack/src/branch/master/README.md">MicroStack documentation</a> or <a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/microstack-get-started#0">Get started with MicroStack tutorial</a>.
-          </p>
-        </div>
+            <p class="p-stepped-list__content u-stepped-list-margin">
+              <a href="https://github.com/CanonicalLtd/multipass/releases" class="p-button--positive js-download"
+                data-os="mac">
+                <span class="p-link--external">Download Multipass for macOS</span>
+              </a>
+            </p>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Start a multipass vm</h3>
+
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" aria-label="sudo snap install microk8s --classic"
+                value="multipass launch --name microkstack-vm --mem 8G --disk 40G" readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
+          </li>
+          <li class="p-stepped-list__item">
+            <h3 class="p-stepped-list__title">Install the MicroStack snap on that multipass vm</h3>
+
+            <div class="p-code-copyable u-stepped-list-margin">
+              <input class="p-code-copyable__input" aria-label="sudo snap install microk8s --classic"
+                value="multipass exec microkstack-vm -- sudo snap install microstack --classic --edge"
+                readonly="readonly">
+              <button class="p-code-copyable__action">Copy to clipboard</button>
+            </div>
+
+            <p class="p-stepped-list__content u-stepped-list-margin">Don’t have the <code>snap</code> command? <a
+                href="https://docs.snapcraft.io/installing-snapd" class="p-link--external">Get set up for snaps.</a></p>
```

Comment:
As above, a new Ubuntu VM will have the `snap` command, so this isn’t necessary.